static/extensions/pooiod/Box2D.js

```diff
@@ -187,24 +185,17 @@ while keeping general compatability. (made with box2D js es6) */
             text: 'Destroy every object',
           },
           {
-            opcode: 'createNoCollideSet',
+            opcode: 'setObjectLayer',
```

Comment:
never change/delete block opcodes

Reply:
The yes/no collide blocks did not get changed or removed. They have been moved down to the "Legacy blocks" section (line 685)